lib/foodcritic/rules/fc121.rb

```diff
@@ -0,0 +1,6 @@
+rule "FC121", "Cookbook depends on cookbook where resources are built into Chef 14" do
```

Comment:
The wording seems to read awkward, but I'm still sick-foggy-brain so not sure if that's me, and don't have a better idea.

Reply:
I need @thommay